.travis.yml

```diff
@@ -23,6 +23,6 @@ jobs:
 script:
  - python --version
  - pip list
- - pytest --cov=src -p no:warnings
+ - pytest --cov=src -p no:warnings -s
```

Comment:
this `-s` can be removed, because of the commented out assertion in `StdioTuple.__new__`.

the alternatives include:
- not creating `_INITIAL_SYS_STREAMS` and re-add the assertion, which requires the caller to disable any incompatible wrappers before using stdio-mgr
- detecting wrappers like pytest, and finding where they hid the real sys streams.
- detecting incompatible wrappers in `__new__` but only emit a warning, and let the user encounter real breakages somewhere else, possibly quite confusing. (this appears to be the `io` approach, as we saw with `buffer=StringIO()` not breaking during instantiation)
- do no checking; user beware.

Reply:
Disabling pytest capturing from within the test suite is given a small note at the bottom of https://docs.pytest.org/en/latest/capture.html .

There was also talk of it being possible using markers https://github.com/pytest-dev/pytest/issues/1599#issue-159286641 

And it seems class scope disabling can be achieved using the pytest plugin manager  https://github.com/pytest-dev/pytest/issues/1599#issuecomment-450738270